docs/user-guide/geoprocessing-tool.md

```diff
@@ -0,0 +1,65 @@
+# GeoProcessing Tool
+
+*******************
+
+This tool allows to perform geographic operations using one or more layers containing geometry and save the result of the operation as a new layer.
+
+!!! note
+    The **GeoProcessing Tool** is not included by default in the MapStore configuration but it can be configured within an [application contexts](application-context.md#configure-plugins) if needed.
+    The plugin works only if the **GeoProcessing WPS service** (available as a community extension) is properly installed in [GeoServer](http://geoserver.org/).
```

Comment:
```suggestion
!!! note
    The **GeoProcessing Tool** is not included by default in the MapStore viewer. It can be configured by editing the default viewer configuration or, from the UI, within an [application contexts](application-context.md#configure-plugins).
    The plugin works only when layers are hosted on a [GeoServer](http://geoserver.org/) where **GeoProcessing WPS service** (available as a community extension) is properly installed in 
```